### ISSUE TITLE
[Feature Request] More customization for notifications

### ISSUE THREAD

**Issue by BOLL7708** [ValveSoftware/openvr#1040]
I was unsure where to put this, the [forums](https://steamcommunity.com/app/250820/discussions/4/) or here, but then I realized I'm doing all my integration through the OpenVR APIs so here I am!

Getting intimate with the notifications I have noticed that the styles for None and Application looks the same, and the ones for Contact_[Status] feel very purpose-made. One expectation I had was for None to at least lose the controller icon. I would have liked no icon as I'm piping Twitch chat messages that has no possible action so there's no reason to go into the dashboard.

Current styles, top to bottom, if my memory serves: None, Application, Contact Active, Contact Disabled, Contact Enabled
![image](https://user-images.githubusercontent.com/12276837/52301195-2a18ba00-298a-11e9-839a-9f3dee4cd32f.png)

While I would love to be able to set a different background I also recognize that a coherent color scheme with Steam blue makes it obvious that it's a system notification and not belonging to the game. 
Still, some variety would be lovely when doing applications with less serious purposes, like the Twitch chat application I'm working on. I would love if hosts, raids, subs, etc could have more differences than just the icon. 

Here are a few Photoshopped samples just to get an idea of how it could look, I can dream right?
![image](https://user-images.githubusercontent.com/12276837/52301206-33098b80-298a-11e9-9067-66b24b07ed82.png)

The purple frame of the NotificationBitmap is from my application, it draws a border on the user avatar using the person's chat color.

**Comment by BOLL7708** [comment]
While I'm on the subject. Considering I'm currently drawing my own colored frame in the bitmap, but there exists styling for the border in the other styles, it would of course be awesome to be able to do even more customization. I'll make a list in case anyone wants to go crazy with this.

1. Background tint (could be presets for legibility)
2. Image border color, just like on the current styles, but custom.
3. Text tints (could be presets for legibility)
4. Visibility and/or customization of the system button instruction graphic, I wouldn't mind replacing it by a chat icon for chat or remove it for more space/smaller notification.
5. A discreet style for unimportant messages, hide the image and controller graphic and crop the area to make it smaller.

**Comment by bddckr** [comment]
Wouldn't it be better to roll your own overlay at this point? Your app can nowadays even turn on the laser beam itself to enable interaction even:
https://github.com/ValveSoftware/openvr/blob/cdc39c0db24ca6b8c725a1c4feaee1e2df89e436/headers/openvr.h#L3335-L3337

**Comment by BOLL7708** [comment]
Maybe, I honestly haven't tried that approach as it has seemed much more involved and notifications did what I was looking for. If they don't change, that's fine, I merely suggest things that could make them even more versatile, in my opinion.

Why I even made my application that pushes notifications is because it places them in my field of view, something custom overlay applications I tried never seemed to bother with. there it was either attached to a fixed place in the world or a tracked device.

Right now people have been asking for my chat application which has had a terrible UI which is why I'm polishing it up, and as such things like these popped up in my mind as I'm working through the Twitch chat APIs. I might look into overlays at some point, maybe when I've finished battling with the input system for my other application 😅
